packages/strapi-plugin-content-type-builder/admin/src/translations/es.json

```diff
@@ -24,29 +80,100 @@
+  "form.attribute.media.allowed-types": "Seleccionar tipos de multimadia permitidos",
+  "form.attribute.media.allowed-types.none": "Ninguno",
+  "form.attribute.media.allowed-types.option-files": "Archivos",
+  "form.attribute.media.allowed-types.option-images": "imágenes",
+  "form.attribute.media.allowed-types.option-videos": "Videos",
+  "form.attribute.media.option.multiple": "Varios archivos Multimedia",
+  "form.attribute.media.option.multiple.description": "Mejor para sliders carruseles o descarga múltiple de archivos",
+  "form.attribute.media.option.single": "Un archivo multimedia",
+  "form.attribute.media.option.single.description": "Mejor para avatares, imagenes de perfil o portadas",
   "form.attribute.settings.default": "Valor por defecto",
+  "form.attribute.text.option.long-text": "Texto largo",
+  "form.attribute.text.option.long-text.description": "Mejor para descripciones, biografías. Las búsquedas exactas no están habilitadas",
+  "form.attribute.text.option.short-text": "Texto corto",
+  "form.attribute.text.option.short-text.description": "Mejor para títulos, nombres y vínculos (URL). La búsqueda exacta por palabra está disponible.",
+  "form.button.add.field.to.collectionType": "Agrega otro campo para este Contenedor",
+  "form.button.add.field.to.component": "Agrega otro campo para este Componente",
+  "form.button.add.field.to.contentType ": "Add another field to this content type",
+  "form.button.add.field.to.singleType": "Agrega otro campo para este Componente Único",
+  "form.button.add-components-to-dynamiczone": "Agrega un componente al área dinámica",
+  "form.button.add-field": "Agreg un nuevo campo",
+  "form.button.add-first-field-to-created-component": "Agrega el primer campo de tu componente",
   "form.button.cancel": "Cancelar",
+  "form.button.collection-type.description": "Mejor para instancias múltiples como artículos, productos, comentarios, etc.",
+  "form.button.configure-view": "Configurar el componente",
+  "form.button.configure-view": "Configurar la vista",
   "form.button.continue": "Continuar",
+  "form.button.delete": "Eliminar",
+  "form.button.finish": "Concluir",
   "form.button.save": "Guardar",
+  "form.button.select-component": "Select un componente",
+  "form.button.single-type.description": "Mejor para instancias únicas como la página de inicio, Quienes Somos, etc.",
   "from": "de",
-  "menu.section.models.name.plural": "Tipos de contenido",
-  "menu.section.models.name.singular": "Tipo de contenido",
+  "injected-components.content-manager.edit-settings-view.link.components": "Editar el componente",
+  "injected-components.content-manager.edit-settings-view.link.content-types": "Editar el Contender",
+  "menu.section.components.name.plural": "Componentes",
+  "menu.section.components.name.singular": "Componente",
+  "menu.section.models.name.plural": "Contenedores",
+  "menu.section.models.name.singular": "Contenedor",
+  "menu.section.single-types.name.plural": "Contenedores Único",
```

Comment:
Forgot to pluralize único: "Únicos"

Reply:
I think will be better ...if is singular "Tipo de contenido" and plural "Tipos de contenido"

Reply:
No, I am not agree, "Tipos de Contenido" has many problems in many places.